cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -1527,4 +1528,17 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
       );
     });
   });
+
+  describe('invalid document structure', () => {
+    describe('normalizable errors', () => {
+      it('does not throw', () => {
+        cy.setInitialValue(invalidDocumentNormalizable);
+        cy.reload();
+
+        //check that editors content is what we expect (not a thrown error)
+        richText.editor.findByText('unordered list first item').should('exist');
+        richText.editor.findByText('unordered list second item').should('exist');
```

Comment:
We should also check there are no validation errors

Reply:
> @IBratWurst Did you try if using `normalizeinitialvalue` prop [would trigger an `onChange` event](https://github.com/contentful/field-editors/blob/4411939b489d1133388c3fa8485ef25069bf1f4c/packages/rich-text/src/RichTextEditor.jsx#L79-L80)? it it doesn't then it would be a cleaner solution

@z0al I tried out `normalizeinitialvalue` (together with initial value) but unfortunately it didn't work. We get the same error as withoutany normalization